mlflow/server/js/src/model-registry/components/ModelPage.js

```diff
@@ -113,8 +113,9 @@ export class ModelPageImpl extends React.Component {
               clearInterval(this.pollIntervalId);
               if (Utils.shouldRender404(requests, [this.initgetRegisteredModelApiId])) {
                 return (
-                  <Error404View
-                    resourceName={`Model ${modelName}`}
+                  <ErrorView
+                    statusCode={404}
+                    resourceName={`Model ${modelName} does not exist`}
```

Comment:
I don't see `resourceName` prop in `ErrorView`?

Reply:
Sorry, it's not `resourceName`, `subMessage` is correct.
Thank you for your careful review :)